### PR TITLE
Fixing documentation

### PR DESCRIPTION
**Pull request by imkin**
Corrected the GOVC Env variables and improved the documentation.

### CODE REVIEW

**Comment by thockin** [comment]
LGTM but you'll need to sign the CLA

**Comment by frapposelli** [comment]
@thockin a Corporate CLA has been signed by VMware and @imkin is correctly added to the google group, see https://github.com/kubernetes/kubernetes/pull/19489#issuecomment-172177111

File: docs/getting-started-guides/vsphere.md [hunk 1]
```diff
@@ -104,10 +104,17 @@ parameters. The guest login for the image that you imported is `kube:kube`.
...
-export KUBERNETES_PROVIDER=vsphere
-cluster/kube-up.sh
+cd kubernetes
+KUBERNETES_PROVIDER=vsphere cluster/kube-up.sh
+```
+
+#### Build from source
+```sh
+cd kubernetes
+make release
```
Comment (vishh) [inline docs/getting-started-guides/vsphere.md]: Could this be `make quick-release`?

**Comment by vishh** [comment]
I can't seem to find @imkin's CLA in the CLA database. Is there a possibility to have used different email addresses?

**Comment by frapposelli** [comment]
@vishh VMware signed a corporate CLA with Google, we don't have individual CLA signed, our google group of authorized committers is vmware-google-contributors@googlegroups.com and @imkin is part of it.

We have been through this already, see https://github.com/kubernetes/kubernetes/pull/19489#issuecomment-172177111

Please advise how to fix this, it's becoming frustrating.

**Comment by vishh** [comment]
@willnorris: @imkin has signed a Corporate CLA as part of VMware, but the CLA system doesn't seem to recognize that. Can you help resolve this issue?

File: docs/getting-started-guides/vsphere.md (hunk 1 above)
Comment (imkin) [inline docs/getting-started-guides/vsphere.md]: Well when we add the e2e tests we want them to run with this I think?

**Comment by willnorris** [comment]
@vishh: I'm tracking down why VMware's CLA is not showing up (you're cc'ed on the email).  In the meantime, you can safely merge this despite what googlebot says.

File: docs/getting-started-guides/vsphere.md (hunk 1 above)
Comment (vishh) [inline docs/getting-started-guides/vsphere.md]: Ok. Its just that `make release` will take quite some time and is not required to be run all the time.

**Comment by vishh** [comment]
Thanks @willnorris! @imkin its now upto the merge bot to pick up this PR. IT should merge it soon given that e2e is not required.

**Comment by vishh** [comment]
@k8s-bot test this github issue #IGNORE

**Comment by frapposelli** [comment]
@willnorris @vishh appreciate the support guys! :+1:

**Comment by imkin** [comment]
@willnorris @vishh thanks for the prompt support on this :-)

File: docs/getting-started-guides/vsphere.md (hunk 1 above)
Comment (imkin) [inline docs/getting-started-guides/vsphere.md]: @vishh agreed I could add both and let the user choose.

**Comment by wojtek-t** [comment]
@imkin 

FAIL: changes needed but not made due to --verify
/home/travis/gopath/src/github.com/kubernetes/kubernetes/docs/ is out of date. Please run hack/update-generated-docs.sh
!!! Error in hack/verify-generated-docs.sh:28
  '"${KUBE_ROOT}/hack/after-build/verify-generated-docs.sh" "$@"' exited with status 1
Call stack:
  1: hack/verify-generated-docs.sh:28 main(...)

Can you please fix?

**Comment by wojtek-t** [comment]
@imkin - you already have writen what to do in this comment:

"Please run hack/update-generated-docs.sh"

**Comment by vishh** [comment]
ok to test

**Comment by imkin** [comment]
@vishh can we tag this for milestone 1.2 ?

**Comment by vishh** [comment]
v1.2 will include this PR since its already merged

**Pull request merged**